libs/agno/agno/workflow/v2/workflow.py

```diff
@@ -420,12 +489,21 @@ def _execute_stream(
                         workflow_run_response=workflow_run_response,
                         step_index=i,
                     ):
+                        # Handle events
+                        if isinstance(event, WorkflowRunResponseEvent):
+                            self._handle_event(event, workflow_run_response)
```

Comment:
Shouldn't you yield this?

Reply:
But that is in the "else"? Or am I looking in the wrong place

Reply:
yes its in else, like we yield all the events but only store/skip the `WorkflowRunResponseEvent`(s)